go.mod

```diff
@@ -52,3 +52,5 @@ require (
 replace github.com/stretchr/objx => github.com/stretchr/objx v0.2.0
 
 replace golang.org/x/crypto => golang.org/x/crypto v0.0.0-20200420201142-3c4aac89819a
+
+replace github.com/digitalocean/godo => github.com/timoreimann/godo v1.7.4-0.20200921135118-82e120c0a69c
```

Comment:
is adding `github.com/timoreimann/godo` just for testing?

Reply:
Yes -- see the PR description about the temporary nature of the associated commit.